backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5709110_sys_gh16561_AD_Printer_Config_UC_Workspace_HostKey.sql

```diff
@@ -0,0 +1,24 @@
+-- 2023-10-29T18:25:14.726Z
+INSERT INTO AD_Index_Table (AD_Client_ID,AD_Index_Table_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,EntityType,ErrorMsg,IsActive,IsUnique,Name,Processing,Updated,UpdatedBy,WhereClause) VALUES (0,540772,0,540637,TO_TIMESTAMP('2023-10-29 19:25:14','YYYY-MM-DD HH24:MI:SS'),100,'D','Pro Arbeitsplatz und Hostkey ist nur ein Datensatz erlaubt','Y','Y','AD_Printer_Config_UC_Workspace_HostKey','N',TO_TIMESTAMP('2023-10-29 19:25:14','YYYY-MM-DD HH24:MI:SS'),100,'IsActive=''Y''')
+;
```

Comment:
'Pro Arbeitsplatz und Hostkey ist nur ein aktiver Datensatz erlaubt.' (IMHO we don't need host key for workplaces)

Reply:
done